### PR TITLE
test(e2e): deneb basic

### PR DESCRIPTION
**Pull request by terencechain**
deneb e2e test

### CODE REVIEW

**Comment by prestonvanloon** [comment]
Fails to build

File: beacon-chain/db/kv/blob.go [hunk 1]
```diff
@@ -47,11 +46,6 @@ func (s *Store) SaveBlobSidecar(ctx context.Context, scs []*ethpb.BlobSidecar) e
 		for k, _ := c.Seek(rotatingBufferPrefix); bytes.HasPrefix(k, rotatingBufferPrefix); k, _ = c.Next() {
 			if len(k) != 0 {
 				replacingKey = k
-				oldSlotBytes := replacingKey[8:16]
-				oldSlot := bytesutil.BytesToSlotBigEndian(oldSlotBytes)
-				if oldSlot >= slot {
-					return fmt.Errorf("attempted to save blob with slot %d but already have older blob with slot %d", slot, oldSlot)
-				}
```
Comment (kasey) [inline beacon-chain/db/kv/blob.go:54]: I know this was a workaround for devnet, but is this behavior we want in prod? Presumably we protect this invariant as part of our circular buffer design, cc @rauljordan

File: beacon-chain/p2p/types/object_mapping.go [hunk 2]
```diff
@@ -32,6 +35,13 @@ var (
 // reset maps and reinitialize them.
 func InitializeDataMaps() {
 	// Reset our block map.
+	log.
+		WithField("genesis_fv", fmt.Sprintf("%#x", params.BeaconConfig().GenesisForkVersion)).
+		WithField("altair_fv", fmt.Sprintf("%#x", params.BeaconConfig().AltairForkVersion)).
+		WithField("bellatrix_fv", fmt.Sprintf("%#x", params.BeaconConfig().BellatrixForkVersion)).
+		WithField("capella_fv", fmt.Sprintf("%#x", params.BeaconConfig().CapellaForkVersion)).
+		WithField("deneb_fv", fmt.Sprintf("%#x", params.BeaconConfig().DenebForkVersion)).
+		Debug("InitializeDataMaps")
```
Comment (kasey) [inline beacon-chain/p2p/types/object_mapping.go:44]: I think I added this while debugging rpc issues, we should remove it.

File: beacon-chain/sync/rpc_chunked_response.go [hunk 3]
```diff
@@ -148,6 +164,37 @@ func extractBlockDataType(digest []byte, tor blockchain.TemporalOracle) (interfa
...
+		log.
+			WithField("valroot", vRoot).
+			WithField("genesis", fmt.Sprintf("%#x", gen)).
+			WithField("altair", fmt.Sprintf("%#x", altair)).
+			WithField("bellatrix", fmt.Sprintf("%#x", bella)).
+			WithField("capella", fmt.Sprintf("%#x", capella)).
+			WithField("deneb", fmt.Sprintf("%#x", deneb)).
+			Debug("context type mapping")
+		ctxByteTableLogged = true
+	}
```
Comment (kasey) [inline beacon-chain/sync/rpc_chunked_response.go:197]: :scissors:

File: go.mod [hunk 4]
```diff
@@ -266,3 +267,5 @@ replace github.com/json-iterator/go => github.com/prestonvanloon/go v1.1.7-0.201
 
 // See https://github.com/prysmaticlabs/grpc-gateway/issues/2
 replace github.com/grpc-ecosystem/grpc-gateway/v2 => github.com/prysmaticlabs/grpc-gateway/v2 v2.3.1-0.20230315201114-09284ba20446
+
+replace github.com/ethereum/go-ethereum => github.com/MariusVanDerWijden/go-ethereum v1.8.22-0.20230626175218-2d586a9714d9
```
Comment (kasey) [inline go.mod:271]: do we want this in `deneb-integration`?

Comment (kasey) [reply go.mod:271]: sgtm

**Review by rauljordan — APPROVED** [review]

**Pull request merged**